__tests__/server/metrics/gauges.spec.js

```diff
@@ -57,24 +58,38 @@ describe('gauges', () => {
 
     it('throws an error if the gauge does not exist', () => {
       const { incrementGauge } = load();
-      expect(() => incrementGauge('nope')).toThrowErrorMatchingSnapshot();
+      expect(() => incrementGauge('nope')).toThrowErrorMatchingInlineSnapshot(
+        '"unable to find gauge nope, please create it first"'
+      );
     });
 
     it('calls the inc method of the gauge', () => {
       const { createGauge, incrementGauge } = load();
-      createGauge({ name: 'yup' });
-      const gauge = Gauge.mock.instances[0];
+      createGauge({ name: 'yup', help: 'yup_help' });
+      const gauge = register._metrics.yup;
+      expect(gauge.hashMap[''].value).toBe(0);
       incrementGauge('yup');
-      expect(gauge.inc).toHaveBeenCalledTimes(1);
+      expect(gauge.hashMap[''].value).toBe(1);
+      incrementGauge('yup', 2);
+      expect(gauge.hashMap[''].value).toBe(3);
     });
 
     it('calls the inc method of the gauge with the arguments', () => {
       const { createGauge, incrementGauge } = load();
-      createGauge({ name: 'yup' });
-      const gauge = Gauge.mock.instances[0];
-      incrementGauge('yup', 1, 'two', [null, null, null]);
-      expect(gauge.inc).toHaveBeenCalledTimes(1);
-      expect(gauge.inc).toHaveBeenCalledWith(1, 'two', [null, null, null]);
+      createGauge({ name: 'yup', help: 'yup_help', labelNames: ['foo'] });
+      const gauge = register._metrics.yup;
+      expect(gauge.hashMap).toEqual({});
+      incrementGauge('yup', { foo: 'bar' }, 2);
```

Comment:
An object of labels (defined at metric creation) is the only other valid argument